srtcore/sfmt.h

```diff
@@ -0,0 +1,1047 @@
+// Formatting library for C++ - C++03 compat version of on-demand tagged format API.
+//
+// Copyright (c) 2024 - present, Mikołaj Małecki

```

Comment:
Copyright - Haivision. Authors: Mikołaj Małecki

Reply:
Well, I'm not sure about that. Although it should be mentioned that it is copied from an open-source library distributed with PD license.